docs/domains/Domain.json

```diff
@@ -101,41 +97,41 @@
           "minimum": 1
         },
         "replicas": {
-          "description": "The number of cluster members to run.",
+          "description": "The number of cluster member Managed Server instances to start. The first cluster members defined in the WebLogic domain configuration will be selected to start, up to the replicas count, unless specific Managed Servers are specified as starting in their entry under the managedServers field. In that case, the specified Managed Servers will be started and then additional cluster members will be started, up to the replicas count, by finding the first cluster members in the WebLogic domain configuration that are not already started. If cluster members are started because of their related entries under managedServers then this cluster may have more cluster members running than its replicas count. Not required. Defaults to 0.",
```

Comment:
We don't use the configuration ordering any more - it's not clear if there is a reliable one available, as returns admin queries were not always in the expected ordere.  Instead, the ordering of instances is alphabetical (or more accurately 'numero lexi' where we canonicalize numbers within strings to a fixed number of digits internally for sorting purposes - this assures 'foo-10' sorts after 'foo-9' etc).